packages/set/src/index.ts

```diff
@@ -18,74 +18,81 @@ const $KEYS = Symbol("track-keys");
 export class ReactiveSet<T> extends Set<T> {
   #triggers = new TriggerCache<T | typeof $KEYS>();
 
-  constructor(values?: Iterable<T> | null) {
+  constructor(values?: readonly T[] | null) {
     super();
```

Comment:
Why not simply use `super(values)`?

Reply:
Not sure I remember the reason, but I think it has sth to do with super(values) overriding all declared properties. All tests are failing if so.

Reply:
I think that values are not added to the triggermap this way. But maybe there is something else too. I just remember it wasn't working without the loop add.